app/javascript/react/pages/NotFoundPage/NotFoundPage.tsx

```diff
@@ -1,7 +1,22 @@
 import React from "react";
+import {
+  NotFoundContainer,
+  NotFoundContent,
+  Title,
+  Description,
+} from "./NotFoundPage.style";
 
-const NotFoundPage = () => {
-  return <div>Page Not Found</div>;
+const NotFoundPage: React.FC = () => {
+  return (
+    <NotFoundContainer>
+      <NotFoundContent>
+        <Title>404 - Page Not Found</Title>
```

Comment:
1. To translations please
2. Will it ever be used for ex. 403 error or any other? If so, it could be adjusted to take number of error as param and give you message that matches the error/ or just generic message and proper error code - only when it is the case 😁

Reply:
Nice, although I think it's overkill for MVP and don't want to spend too much time as it doesn't bring that much value ;D  I'll change it to just "Page not found", I wrote it as something generic but you're totally right, I shouldn't use invalid error code 🤡